ansible/roles/test/tasks/bgp_speaker.yml

```diff
@@ -213,5 +213,5 @@
       with_items: "{{vlan_ips}}"
 
     - name: Remove Assigned IPs
-      shell: ip addr flush dev eth{{ '%d' % (minigraph_vlans[minigraph_vlan_interfaces[0]['attachto']]['members'][0] | replace("Ethernet", "") | int / 4)}}
+      shell: ip addr flush dev eth{{ '%d' % (minigraph_port_indices[minigraph_vlans[minigraph_vlan_interfaces[0]['attachto']]['members'][0]])}}
```

Comment:
The start.sh script is generated from template start.j2.
In start.j2 template, the interface is not hardcoded. The interface name is selected using method `eth{{ '%d' % (minigraph_port_indices[minigraph_vlans[minigraph_vlan_interfaces[0]['attachto']]['members'][0]])}}`
So, for flushing, it would be better to use the same method to select the interface.